sdk/eventhub/Azure.Messaging.EventHubs/src/PartitionReceiver.cs

```diff
@@ -1,251 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Threading;
-using System.Threading.Tasks;
-using Azure.Core;
-using Azure.Messaging.EventHubs.Core;
-using Azure.Messaging.EventHubs.Metadata;
-
-namespace Azure.Messaging.EventHubs
-{
-    /// <summary>
-    ///   Allows events from a specific partition of an Event Hub, and in the context
-    ///   of a specific consumer group, to be read with a greater level of control over
-    ///   communication with the Event Hubs service than is offered by other event consumers.
-    /// </summary>
-    ///
-    /// <remarks>
-    ///   It is recommended that the <see cref="EventProcessorClient" /> or <see cref="EventHubConsumerClient" />
-    ///   be used for reading and processing events for the majority of scenarios.  The partition receiver is
-    ///   intended to enable scenarios with special needs which require more direct control.
-    /// </remarks>
-    ///
-    /// <seealso cref="EventProcessorClient" />
-    /// <seealso cref="EventHubConsumerClient.ReadEventsFromPartitionAsync(string, EventPosition, CancellationToken)"/>
-    /// <seealso cref="EventHubConsumerClient.ReadEventsFromPartitionAsync(string, EventPosition, TimeSpan?, CancellationToken)"/>
-    ///
-    public class PartitionReceiver : IAsyncDisposable
```

Comment:
Because it is likely to change form; I have the code split off to the side for now.   I fear that we've been marking too many things internal and are likely to lose track.